javascript/javascript1/week3/homework5.js

```diff
@@ -0,0 +1,132 @@
+// CactusIO-interactive (Smart phone usage app) optional
+
+// Adding an activity
+
+const activities = [];
+/**
+ *
+ *
+ * @param {string} date
+ * @param {string} activity
+ * @param {number} duration
+ */
+const addActivity = (date, activity, duration) => {
+  activities.push({
+    date: date,
+    activity: activity,
+    duration: duration,
+  });
+};
+addActivity("23/7-18", "Youtube", 30);
+
+// Show my status
+
+// This function should use the activities variable and return
+// a string saying the following: "You have added 3 activities. They amount to 78 min. of usage"
+
+const usageLimit = 60;
+
+const showStatus = (arr) => {
+  let minutesOfUsage = 0;
+
+  for (let i = 0; i < arr.length; i++) {
+    minutesOfUsage += arr[i].duration;
+  }
+
+  if (minutesOfUsage >= usageLimit) {
+    return console.log(
+      "You have reached your limit, no more smartphoning for you!"
+    );
+  } else {
+    return console.log(
+      arr.length !== 0
+        ? `You have added ${arr.length} activities. They amount to ${minutesOfUsage} min. of usage`
+        : `Add some activities before calling showStatus`
+    );
+  }
+};
+console.log("\x1b[35m%s\x1b[0m", "\nShowstatus");
+showStatus(activities);
+
+// Usage limit
+
+// Improved addActivity, so that we dont need to specify the date,
+const date = new Date();
+const options = {
+  weekday: "long",
+  year: "numeric",
+  month: "long",
+  day: "numeric",
+};
+const dateToday = date.toLocaleDateString("en-EN", options);
```

Comment:
new Date() would suffice but it's good to be aware of the various new Date() methods. They will come in handy in the future